vizro-core/src/vizro/models/_navigation/_accordion.py

```diff
@@ -26,17 +26,17 @@ class Accordion(VizroBaseModel):
     _validate_pages = validator("pages", allow_reuse=True, always=True)(_validate_pages)
 
     @_log_call
-    def build(self):
+    def build(self, page_id):
         if self.pages:
-            return self._create_accordion()
-        return self._create_default_accordion()
+            return self._create_accordion(page_id=page_id)
+        return self._create_default_accordion(page_id=page_id)
 
-    def _create_accordion_buttons(self, accordion_pages):
+    def _create_accordion_buttons(self, accordion_pages, page_id):
         return [
             dbc.Button(
                 children=[page["name"]],
                 key=page["relative_path"],
-                className="accordion-item-button",
+                className="accordion-item-button-active" if page_id == page["name"] else "accordion-item-button",
```

Comment:
Instead of switching between different CSS classes, we can just use the existing class and leverage its state. For this you need to use the active parameter.

```suggestion
                className="accordion-item-button",
                active=True if page_id == page["name"] else False,
```

I'll comment on the accordion.css file how you would have to change the CSS properties there accordingly